CHANGELOG.md

```diff
@@ -34,6 +34,10 @@ Using the following categories, list your changes in this order:
 
 ## [Unreleased]
 
+### Removed
+
+-   `django-reactpy` database entries are no longer cleaned during Django application startup. Instead, it will occur on webpage loads, only when database after `REACTPY_RECONNECT_MAX` seconds has elapsed since the last cleaning.
```

Comment:
```suggestion
-   `django-reactpy` database entries are no longer cleaned during Django application startup. Instead, it will occur on webpage loads if `REACTPY_RECONNECT_MAX` seconds has elapsed since the last cleaning.
```